src/lib/components/Swap/Output.tsx

```diff
@@ -101,6 +126,11 @@ export default function Output({ disabled, children }: OutputProps) {
           </ThemedText.Body2>
         </TokenInput>
         {children}
+        <UniswapA href={`https://app.uniswap.org/`}>
+          <Logo />
+          &nbsp;
```

Comment:
Instead of a non-blocking space, use a `Row` with a `gap` so you have obvious control over the spacing between the logo and the text:

```
`<Row gap={0.5}><Logo /><Trans>Powered by...</Trans></Row>
```